src/patterns/ec2-app/base.ts

```diff
@@ -143,6 +143,8 @@ export interface GuEc2AppProps extends AppIdentity {
   certificateProps: GuDomainName;
   withoutImdsv2?: boolean;
   imageRecipe?: string;
+  vpcOverride?: IVpc;
+  deploymentSubnetsOverride?: ISubnet[];
```

Comment:
I think `vpc` and `privateSubnets` would be clearer naming here, as the type of `undefined | T` already suggests it's possible to override any defaulted value.

Reply:
OK, fair enough. I shied away from naming it "vpc" due to a naming collision in the constructor function but that is an easy change